app/code/Magento/CatalogGraphQl/etc/schema.graphqls

```diff
@@ -49,6 +49,12 @@ enum PriceTypeEnum @doc(description: "This enumeration the price type.") {
     DYNAMIC
 }
 
+enum CustomizableDateTypeEnum @doc(description: "This enumeration customizable date type.") {
+    date
```

Comment:
these should be all caps
I think you need a resolver for that or en enum value mapper converter

Reply:
@cpartica, thanks for the review.
Changes were added.